lib/features.js

```diff
@@ -152,7 +152,7 @@ const features = {
         method: 'enableStimulusBridge()',
         packages: [
             { name: '@symfony/stimulus-bridge' },
-            { name: 'stimulus' }
+            { name: 'stimulus', enforce_version: true }
```

Comment:
the commit message talks about bumping stimulus version, but that's not what you bumped

Reply:
yup - this was a mistake - it should be on the other one - thanks for catching!